src/System.Text.Json/tests/Serialization/EnumConverterTests.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System.IO;
+using System.Runtime.InteropServices.ComTypes;
```

Comment:
I don't see anything that would have come from this namespace... do we need this using-import?

Reply:
Weird. Not sure how that happened.